### PR TITLE
Fix typos and broken links on docs 

### LINKED ISSUE #1821: Docs and tutorials typos
### Problem

Fix the following list of typos / broken links

1. https://docs.onflow.org/cadence/language/transactions/#summary
 "Signing accounts can't be accesed anywhere else in the transaction."
1. https://docs.onflow.org/cadence/tutorial/04-capabilities/ 
Transaction1.cdc has a comment on the prepare section that seemed to be from a previous version. It says “no need to do anything in prepare”, but in this case, things are being done in the prepare section
In Transaction3.cdc - “The capability is created and stored at /public/Hello” - actually /public/HelloAssetTutorial
In Script1.cdc, optional chaining is mentioned in a comment, but there doesn't seem to be optional chaining where the comment is
1. https://docs.onflow.org/cadence/tutorial/02-hello-world/
The code in Playground uses "access(all)" but the documentation uses "pub". A few lines down, it's explained that the two are interchangeable, but as a reader I was confused or thought there was a mistake until I got to that explanation. 
1. https://docs.onflow.org/cadence/tutorial/05-non-fungible-tokens-2/#mint-and-distribute-tokens
Just before the section "Transferring an NFT", there is a code snippet that is the output of a script. It says "Account 2 NFTs" but I think it's intended to be Account 1. 
At the bottom of section 5, it says the next section is Marketplaces, but it is currently Fungible tokens
1. https://docs.onflow.org/cadence/tutorial/06-fungible-tokens/
There's a broken image near the text "This deployment stores the contract for the"
"one of THE MOST COMMON security mistake" - "mistakes"?
0x02 contract has typo: “   // accesible by the owner of the account”
Get Balances Script: mentions optional chaining, but there doesn't seem to be optional chaining on the commented line
At the bottom, the next section mentioned is non-fungible tokens, but that was the previous section.
1. https://docs.onflow.org/cadence/tutorial/08-marketplace-compose/
Broken link: "[We have covered capabilities in previous tutorials(06-fungible-tokens#ensuring-security-in-public-capability-security)], but only the basics. Capabilities can be used for so much more!"
1. https://docs.onflow.org/cadence/design-patterns/
LockedTokens.cdc link to init() should be https://github.com/onflow/flow-core-contracts/blob/master/contracts/LockedTokens.cdc#L718
1. https://docs.onflow.org/cadence/anti-patterns/
"solenFunds" stolenFunds?
1. https://docs.onflow.org/cadence/tutorial/10-resources-compose/
Transaction1 has errors that prevent deploying
1. https://docs.onflow.org/cadence/tutorial/09-voting/
"The admin calls this function to create a new Ballo" - Ballot?
Transaction 3 Cast Vote is empty

### PR DESCRIPTION
**Pull request by alilloig**
Closes #1821

## Description

Fix some typos, outdated and broken images and broken links we knew about thanks to some feedback

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/tutorial/05-non-fungible-tokens-2.mdx [hunk 1]
```diff
@@ -673,10 +673,9 @@ for information about the official Flow NFT standard and an example implementati
...
-## Create a Flow Marketplace
+## Fungible Tokens
 
 ---
 
-Now that you have a working NFT, you can attempt to extend its functionality on your own,
-or you can learn how to create a marketplace that uses both fungible tokens and NFTs.
-Move on to the next tutorial to learn about Marketplaces in Cadence!
+Now that you have a working NFT, you will probably want to be able to trade it. For that you are going to need to
+understand how fungible tokens works on Flow, so go ahead and move to the next tutorial!
```
Comment (turbolent) [inline docs/tutorial/05-non-fungible-tokens-2.mdx:681]:
```suggestion
understand how fungible tokens work on Flow, so go ahead and move to the next tutorial!
```

File: docs/tutorial/06-fungible-tokens.mdx [hunk 2]
```diff
@@ -1111,8 +1111,8 @@ From here, you could try to extend the functionality of fungible tokens by makin
 - An escrow that can be deposited to (but only withdrawn when the balance reaches a certain point)
 - A function to the resource that mints new tokens!
 
-## Non-Fungible Tokens on Flow
+## Create a Flow Marketplace
 
 ---
-
-Now that you have an understanding of how fungible tokens work on Flow, you're ready to play with non-fungible tokens!
+Now that you have an understanding of how fungible tokens work on Flow and a working NFT, you can learn how to create
```
Comment (turbolent) [inline docs/tutorial/06-fungible-tokens.mdx:1117]:
```suggestion
Now that you have an understanding of how fungible tokens work on Flow and have a working NFT, you can learn how to create 
```

**Review by turbolent — APPROVED** [review]

**Review by joshuahannan — APPROVED** [review]
Did you just create your own playground links with the fixed code?

**Comment by alilloig** [comment]
> Did you just create your own playground links with the fixed code?

Yeah exactly, I forked the projects and fix the things there, it's that ok?

**Comment by joshuahannan** [comment]
yeah, I just wanted to make sure

**Pull request merged**